ktor-server/ktor-server-plugins/ktor-server-auth/jvm/src/io/ktor/server/auth/Authentication.kt

```diff
@@ -162,7 +162,7 @@ public class Authentication private constructor(config: Configuration) {
 
         override val key: AttributeKey<Authentication> = AttributeKey("Authentication")
 
-        override fun install(pipeline: Application, configure: Configuration.() -> Unit): Authentication {
+        override fun install(application: Application, pipeline: Application, configure: Configuration.() -> Unit): Authentication {
```

Comment:
While briefly looking at the new implementation, I didn't notice why do we need this parameter now?